Code/Framework/AzNetworking/AzNetworking/UdpTransport/UdpNetworkInterface.cpp

```diff
@@ -125,8 +125,15 @@ namespace AzNetworking
         }
     }
 
-    ConnectionId UdpNetworkInterface::Connect(const IpAddress& remoteAddress)
+    ConnectionId UdpNetworkInterface::Connect(const IpAddress& remoteAddress, uint16_t localPort)
     {
+        if (m_connectionSet.GetConnection(remoteAddress) != nullptr)
+        {
+            AZLOG_INFO("Attempting to connect to an endpoint that already has a connection");
```

Comment:
Minor: Log remote address to make troubleshooting easier?

Reply:
+1